src/custom_OSCP_report.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
A blank template is not needed.

Reply:
fair enough. Figured it would be convenient for those like me who tweaked existing formats into their own template, but that's valid. Thanks for the review.

Reply:
Thanks for your interest. I plan to create an OSWE template, but if you want to submit an OSCE or OSEE, etc. template, feel free to do it.